react-delegationdashboard/src/components/UndelegateModal/index.tsx

```diff
@@ -34,6 +33,10 @@ const UndelegateModal = ({ show, title, description, handleClose, handleContinue
             }}
             validationSchema={object().shape({
               amount: string().required('Required')
```

Comment:
I think we can pull the validation outside of the component, it would be cleaner to have: `validationSchema={ myStuff }`